scripts/lib/setup_venv.py

```diff
@@ -105,6 +105,7 @@ def install_venv_deps(pip, requirements_file):
     # type: (str, str) -> None
     pip_requirements = os.path.join(ZULIP_PATH, "requirements", "pip.txt")
     run([pip, "install", "-U", "--requirement", pip_requirements])
+    run([pip, "install", "Cython==0.29.7"])
```

Comment:
What's the story with this?  I don't see why we'd need to install Cython manually...

Reply:
The problem here is, since line_profiler doesn't list cython in setup_requires or pyproject.toml, pip will skip the .pyx files and used the precompiled c files as cython is not available at build time. So it has to be installed before trying to install line_profiler. Using the PR mentioned above will solve this problem

Reply:
Yeah, so let's plan to not do this and instead do one of the other solutions for line_profiler (either getting them to do a fixed release or building off a fork containing that PR).